packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts

```diff
@@ -268,6 +268,19 @@ export class PluginViewRegistry implements FrontendApplicationContribution {
                 }
             }
         }));
+
+        toDispose.push(this.commands.registerCommand({
```

Comment:
Is it something to align with VS Code? Otherwise we already have commands to toggle and open views. I don't think we need more.

Reply:
Exactly. when a View is Registered in VSCode a `Focus` command for such a view is registered automatically. This is a standard VSCode behaviour. 
In case of VSCode Reference View` plugin this `Focus` command is used to open/activate the References View on execution of `Find All References` and `Find All Implementations` commands. Without having  this command added an error is throws as well as the View is not activated. In the plugin it is used here:
https://github.com/microsoft/vscode-references-view/blob/master/src/extension.ts#L23